models/tenant.rb

```diff
@@ -0,0 +1,20 @@
+ruby_tenants = []
+tenants = data[:tenants]
+tenants.each do |tenant|
+  puts tenant
+
+class Tenant
+  def initialize( address, rent, age, apartment_id)
```

Comment:
how can we access properties of a tenant currently? the `initialize` method only sets them initially, in order to actually read these properties or reset existing one, we need getter/setter methods (re: [`attr_accessor`](http://stackoverflow.com/questions/4370960/what-is-attr-accessor-in-ruby)